components/dot_menu/index.ts

```diff
@@ -127,11 +128,11 @@ function mapStateToProps(state: GlobalState, ownProps: Props) {
         components: state.plugins.components,
         postEditTimeLimit: config.PostEditTimeLimit,
         isLicensed: license.IsLicensed === 'true',
-        teamId: getCurrentTeamId(state),
+        teamId: channel.team_id,
```

Comment:
I think this might cause an issue with the Add Reactions option [here](https://github.com/mattermost/mattermost-webapp/blob/84417ddbb12d51f1c4ddbc73c35ee6fdceb3ab18/components/dot_menu/dot_menu.tsx#L496) since `teamId` will be empty now for DMs/GMs.

Maybe we could just remove the `currentTeamId` prop entirely and just have this be `channel.team_id || currentTeam.id`?